jdi-light-mobile-tests/src/main/java/io/github/com/pages/LogSidebar.java

```diff
@@ -0,0 +1,9 @@
+package io.github.com.pages;
+
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+
+public class LogSidebar {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'LogSidebar' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)